src/UXClient/Components/Marker/Marker.scss

```diff
@@ -14,6 +14,16 @@
     .tsi-markerTimeLabel {
         background-color: $gray3;
         border-color: $gray6;
```

Comment:
also maybe $gray2?  like that light blue color in light theme?  then we refactor the x and y tooltips for the guidline things to be divs and make em look like this with slightly higher z index